test/lib.bash

```diff
@@ -92,6 +93,9 @@ function downstream_serving_e2e_tests {
   return $failed
 }
 
+function downstream_knative_kafka_e2e_tests {
+}
```

Comment:
Lacks the fix for this I fear, so I guess it'll stumble before even trying to compile? :thinking:

Reply:
aw man, crap!

Reply:
Too many balls in the air? :juggling_person:

Reply:
thanks. rebased now!